coverage/plugin_support.py

```diff
@@ -51,19 +53,23 @@ def load_plugins(
         plugins = cls()
         plugins.debug = debug
 
-        for module in modules:
-            plugins.current_module = module
-            __import__(module)
-            mod = sys.modules[module]
+        if plugin_override is not None:
+            for override in plugin_override:
+                override(plugins, {})
```

Comment:
Of course, after merging, thinking about documenting this, I have a hard time explaining why config options are passed to this function but they are always empty.  I'm going to change the signature of the plugin function to leave it out.

Reply:
Ok! I went for consistency with the signature of plugin inits loaded from the config, but that did produce this slightly silly result.